### PR TITLE
Added SCLI-21, SCLI-22, SCLI-37 

### PR DESCRIPTION
**Pull request by metinavc1**
SCLI-21 Ability to suppress Merchant Password from POS for refund
SCLI-22 Allow surcharging from POS
SCLI-37 Automated Address Resolution - Implement device lookup for IP address using Serial Number

### CODE REVIEW

File: ramenpos/Podfile [hunk 1]
```diff
@@ -6,7 +6,9 @@ target 'RamenPos' do
     use_frameworks!
     
     # Pods for RamenPos
-    pod 'SPIClient-iOS', '~> 2.4.0'
+    #pod 'SPIClient-iOS', '~> 2.4.4'
+    pod 'SPIClient-iOS', :git => 'https://github.com/AssemblyPayments/spi-client-ios', :branch => 'feature/v2.4.x'
```
Comment (chrishulbert) [inline ramenpos/Podfile]: Is this supposed to be checked in? Just checking :)

File: ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift [hunk 2]
```diff
@@ -175,6 +175,7 @@ extension MainViewController {
             logMessage(String(format: "# CASHOUT: %.2f", Float(cashoutResponse.getCashoutAmount()) / 100.0))
             logMessage(String(format: "# BANKED NON-CASH AMOUNT: %.2f", Float(cashoutResponse.getBankNonCashAmount()) / 100.0))
             logMessage(String(format: "# BANKED CASH AMOUNT: %.2f", Float(cashoutResponse.getBankCashAmount()) / 100.0))
+            logMessage(String(format: "# SURCHARGE: %.2f", Float(cashoutResponse.getSurchargeAmount()) / 100.0))
```
Comment (chrishulbert) [inline ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift:178]: Typically financial amounts should be in a Double for accuracy but no big deal here it's just a sample :)

**Review by chrishulbert — APPROVED** [review]
Looks good 👍

File: ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift (hunk 2 above)
Comment (metinavc1) [reply ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift:178]: Ok i got it. Thanks

File: ramenpos/Podfile (hunk 1 above)
Comment (metinavc1) [reply ramenpos/Podfile]: After releasing I will change it

**Pull request merged**